docs/guide/pixel_math.md

```diff
@@ -210,10 +210,10 @@ Visually, in bits:
 This provides us with an increasing index, that will not overlap
 between spatially partitioned data files.
 
-### compute_hipscat_id
+### compute_hats_id
```

Comment:
Given that you aren't renaming hipscat_id yet, do you want to leave it here in the documentation as well?